gdc/git.go

```diff
@@ -216,7 +270,7 @@ func getFullSHA(shortSHA string) (fullSHA string) {
 
 	switch n := len(results); n {
 	case 0:
-		errTxt := fmt.Sprintf("No SHA found starting with %s", shortSHA)
+		errTxt := fmt.Sprintf("No SHA/branch/tag found like \"%s\"", shortSHA)
```

Comment:
Just FYI - the `%q` formatter is handy for encoding strings. It adds the double-quotes, and safely encodes the contents.
https://golang.org/pkg/fmt/
> ```
> %q	a double-quoted string safely escaped with Go syntax
> ```

Reply:
Very handy indeed, I've added it